tests/builders/metrics_file_regex_with_invalid_linenum.yml

```diff
@@ -0,0 +1 @@
+/Users/sean/Documents/LBNL/code/buildtest/tutorials/metrics/metrics_file_regex_with_invalid_linenum.yml
```

Comment:
please fix symlink

Reply:
you should use relative paths to create symlinks take a look below

```
☁  builders [lsf_queue_check_in_executors] ⚡  ls -l
total 40
-rw-r--r--@ 1 siddiq90  siddiq90     0 Mar 20 14:41 __init__.py
lrwxr-xr-x@ 1 siddiq90  siddiq90    41 Mar 20 14:41 assert_eq.yml -> ../../tutorials/perf_checks/assert_eq.yml
lrwxr-xr-x@ 1 siddiq90  siddiq90    41 Mar 20 14:41 assert_ge.yml -> ../../tutorials/perf_checks/assert_ge.yml
lrwxr-xr-x@ 1 siddiq90  siddiq90    41 Mar 20 14:41 assert_gt.yml -> ../../tutorials/perf_checks/assert_gt.yml
```

Reply:
👍